commit 6e09bef3cbe2c0ad2f387ce1dbfd877c97d6a029
Author: Heiner Kallweit <hkallweit1@gmail.com>
Date:   Mon Jan 7 21:49:09 2019 +0100

    r8169: load Realtek PHY driver module before r8169
    
    [ Upstream commit 11287b693d03830010356339e4ceddf47dee34fa ]
    
    This soft dependency works around an issue where sometimes the genphy
    driver is used instead of the dedicated PHY driver. The root cause of
    the issue isn't clear yet. People reported the unloading/re-loading
    module r8169 helps, and also configuring this soft dependency in
    the modprobe config files. Important just seems to be that the
    realtek module is loaded before r8169.
    
    Once this has been applied preliminary fix 38af4b903210 ("net: phy:
    add workaround for issue where PHY driver doesn't bind to the device")
    will be removed.
    
    Fixes: f1e911d5d0df ("r8169: add basic phylib support")
    Signed-off-by: Heiner Kallweit <hkallweit1@gmail.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/ethernet/realtek/r8169.c b/drivers/net/ethernet/realtek/r8169.c
index b4bca27b460f..07f3080eca18 100644
--- a/drivers/net/ethernet/realtek/r8169.c
+++ b/drivers/net/ethernet/realtek/r8169.c
@@ -717,6 +717,7 @@ module_param(use_dac, int, 0);
 MODULE_PARM_DESC(use_dac, "Enable PCI DAC. Unsafe on 32 bit PCI slot.");
 module_param_named(debug, debug.msg_enable, int, 0);
 MODULE_PARM_DESC(debug, "Debug verbosity level (0=none, ..., 16=all)");
+MODULE_SOFTDEP("pre: realtek");
 MODULE_LICENSE("GPL");
 MODULE_FIRMWARE(FIRMWARE_8168D_1);
 MODULE_FIRMWARE(FIRMWARE_8168D_2);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1029_6e09bef3cbe2c0ad2f387ce1dbfd877c97d6a029.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1029_6e09bef3cbe2c0ad2f387ce1dbfd877c97d6a029.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
